app/controllers/api/account.php

```diff
@@ -1072,39 +1107,41 @@
 
         $queueForEvents->setPayload(
             $response->output(
-                $token->setAttribute('secret', $loginSecret),
+                $token->setAttribute('secret', $tokenSecret),
                 Response::MODEL_TOKEN
             )
         );
 
         // Hide secret for clients
-        $token->setAttribute('secret', ($isPrivilegedUser || $isAppUser) ? $loginSecret : '');
+        $token->setAttribute('secret', ($isPrivilegedUser || $isAppUser) ? $tokenSecret : '');
 
         $response
             ->setStatusCode(Response::STATUS_CODE_CREATED)
             ->dynamic($token, Response::MODEL_TOKEN)
         ;
     });
 
-App::put('/v1/account/sessions/magic-url')
-    ->desc('Create magic URL session (confirmation)')
-    ->groups(['api', 'account', 'session'])
-    ->label('scope', 'public')
+App::put('/v1/account/sessions/token')
+    ->alias('/v1/account/sessions/magic-url')
+    ->alias('/v1/account/sessions/phone')
+    ->desc('Exchange token for session')
     ->label('event', 'users.[userId].sessions.[sessionId].create')
-    ->label('audits.event', 'session.update')
+    ->groups(['api', 'account'])
+    ->label('scope', 'sessions')
+    ->label('auth.type', 'token')
+    ->label('audits.event', 'session.create')
     ->label('audits.resource', 'user/{response.userId}')
     ->label('audits.userId', '{response.userId}')
     ->label('usage.metric', 'sessions.{scope}.requests.create')
-    ->label('usage.params', ['provider:magic-url'])
     ->label('sdk.auth', [])
     ->label('sdk.namespace', 'account')
-    ->label('sdk.method', 'updateMagicURLSession')
-    ->label('sdk.description', '/docs/references/account/update-magic-url-session.md')
-    ->label('sdk.response.code', Response::STATUS_CODE_OK)
+    ->label('sdk.method', ['exchangeTokenForSession', 'updateMagicURLSession', 'updatePhoneSession'])
```

Comment:
`exchangeTokenForSession` doesnt follow our patterns. I think `createSession ` would make more sense.

Reply:
Slightly concerned about this as we already have `createSession` on the users API